compiler/rustc_target/src/spec/targets/wasm32v1_none.rs

```diff
@@ -0,0 +1,88 @@
+//! A "bare wasm" target representing a WebAssembly output that makes zero
```

Comment:
All of this context and explanation is great, it should go as well (or entirely) into a new entry for the target in the `rustc` book:
 - needs to be in the "Tier 2 without host" table, https://github.com/rust-lang/rust/blob/df1b5d3cc2117f1ee96abca25678bc5f5604d450/src/doc/rustc/src/platform-support.md?plain=1#L136
 - with a new file in https://github.com/rust-lang/rust/tree/master/src/doc/rustc/src/platform-support following the [`TEMPLATE`](https://github.com/rust-lang/rust/blob/df1b5d3cc2117f1ee96abca25678bc5f5604d450/src/doc/rustc/src/platform-support/TEMPLATE.md)
 - as well as not forgetting to add the newly created page inside the [`SUMMARY.md`](https://github.com/rust-lang/rust/blob/df1b5d3cc2117f1ee96abca25678bc5f5604d450/src/doc/rustc/src/SUMMARY.md)

*Don't hesitate to take a look at the other wasm targets platform entries[^1][^2].*

[^1]: https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/wasm32-wasip1.md
[^2]: https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/wasm32-wasip1-threads.md

Reply:
Will do, thanks!

Reply:
That will also take care of tidy's whining.

Reply:
Done.